ConcernsCaseWork/ConcernsCaseWork/Models/TrustDetailsModel.cs

```diff
@@ -9,8 +11,14 @@ public sealed class TrustDetailsModel
 	{
 		public GiasDataModel GiasData { get; }
 		
-		public List<EstablishmentModel> Establishments { get; } 
-		
+		public List<EstablishmentModel> Establishments { get; }
+
+		public double TotalPupilCapacity { get { return this.Establishments.Sum(establishment => int.Parse(establishment.SchoolCapacity)); } }
```

Comment:
Can we create a separate method that parses the string in case of an exception returns 0, would avoid blowing up the page.